app/views/passengers/index.haml

```diff
@@ -0,0 +1,57 @@
+= form_tag(passengers_path, :method => "get") do
+  = label_tag 'Search by name:'
+  = text_field_tag :search, @search_term, placeholder: 'Enter a name'
+  %p
+  = label_tag 'Filter by Type:'
+  Permanent
+  = radio_button_tag :filter, 'permanent'
+  Temporary
+  = radio_button_tag :filter, 'temporary'
+  Show All
+  = radio_button_tag :filter, 'all'
+  = submit_tag "Find passengers"
+%p
+= link_to "Add New Passenger", new_passenger_path
+
+- if @permanent.present?
+  %p
+    %b Showing all Permanent
+- elsif @temporary.present?
+  %p
+    %b Showing all Temporary
+- else
+  %p
+    %b Showing All Passengers
+
+- if @passengers.present?
+  %table
+    %thead
+      %b
+      %tr
+        %th Name:
+        %th Email:
+        %th Phone:
+        %th Active:
+        %th Permanent:
+        %th
+        %th
+        %th
+    %tbody
+      - @passengers.each do |passenger|
+        %tr
+          %td= passenger.name
+          %td= passenger.email
+          %td= passenger.phone
+          - if passenger.active
```

Comment:
For boolean columns, it's nice to query them using a predicate method (here, `active?`). This means you don't have to worry about defining a third behavior for when it might be `nil`. Predicate methods coerce that to `false`.